src/hub.js

```diff
@@ -214,18 +215,36 @@ function mountUI(props = {}) {
   const disableAutoExitOnConcurrentLoad = qsTruthy("allow_multi");
   const forcedVREntryType = qs.get("vr_entry_type");
 
+  const Router =
+    process.env.RETICULUM_SERVER && process.env.RETICULUM_SERVER !== document.location.host
+      ? HashRouter
+      : BrowserRouter;
```

Comment:
I remember some nightmares with Android webviews and the BrowserRouter. Probably fine now, but worth giving extra care to testing Oculus Go.

Reply:
note this is just for dev